addons/note/static/src/scss/note.scss

```diff
@@ -37,7 +42,7 @@
             min-height: 300px;
         }
     }
-    .oe_form_field.oe_memo {
+    .oe_memo, .oe_form_field {
```

Comment:
This is surprising. What are the other fields that are specific to Note, are not `oe_memo`, and for which we want those styles applied ?

Reply:
The `oe_memo` class used to be on the `.oe_form_field` but now `.oe_form_field` is the first child of `.oe_memo`. We need to apply this to both of them. I don't know why an extra level was added but that's the consequence.